test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -666,7 +666,7 @@ TEST(nntrainer_capi_nnmodel, getLayer_04_n) {
   EXPECT_EQ(status, ML_ERROR_NONE);
 
   status = ml_train_model_get_layer(model, "inputlayer", &get_layer);
-  EXPECT_EQ(status, ML_ERROR_NOT_SUPPORTED);
+  EXPECT_EQ(status, ML_ERROR_NONE);
```

Comment:
This makes getLayer_04_n as not a negative test any more.

Reply:
%s/n/p done. thanks.

Reply:
Modified unittest getLayer_04_p has no different with getLayer_02_p.
I think we can remove this unittest.